src/main/java/com/checkmarx/flow/config/FlowProperties.java

```diff
@@ -41,6 +41,7 @@
     private Integer httpConnectionTimeout = 30000;
     private Integer httpReadTimeout = 120000;
     private boolean listFalsePositives = false;
+    private Boolean scanResubmit = false;
```

Comment:
Can it be 'boolean' instead of 'Boolean'?

Reply:
Replaced with boolean.